alf/optimizers/trusted_updater.py

```diff
@@ -15,11 +15,13 @@
 
 from typing import Callable
 
-import tensorflow as tf
+import torch
+import torch.nn as nn
 
+import alf
 from alf.utils import math_ops
 
-nest_map = tf.nest.map_structure
+nest_map = alf.nest.map_structure
 
 
 class TrustedUpdater(object):
```

Comment:
Needs to inherit nn.Module to be checkpoint-able?
Also add test to check state_dict of this object?

Reply:
This needs not be checkpointed.

Reply:
TrustedUpdater is created in calc_loss(). At that point, it can get the correct prev_vars.